lib/matplotlib/scale.py

```diff
@@ -100,8 +107,8 @@ def set_default_locators_and_formatters(self, axis):
 
     def get_transform(self):
         """
-        The transform for linear scaling is just the
-        :class:`~matplotlib.transforms.IdentityTransform`.
+        Return the transform for linear scaling, which is just the
```

Comment:
just inherit all these docstrings? not sure per-scale documentation of get_transform is really useful.

Reply:
For now, I'd leave it. There's a bit of value to have the link to the respective transform in the subclasses. Alternatively, one could document these relations for the builtin scales in a common place. and eliminate most of the `get_transform`s. Almost all are just returning `self._transform`. But that's for another time. You are welcome to make a PR (*after* this is merged :wink:).